packages/components/src/components/modal/modal.tsx

```diff
@@ -63,8 +94,8 @@ const ModalElement = ({
         );
     };
 
-    const closeModal = e => {
-        if (is_open) toggleModal(e);
+    const closeModal = (e: React.MouseEvent<HTMLElement>) => {
```

Comment:
```suggestion
    const closeModal = (e: React.MouseEvent<HTMLDivElement>) => {
```

Reply:
We can’t be sure, that outside clicked element is a 'div', so it’s better to use more common type